wikimedia/wikipedia-ios
# Calculate maximum amount donation for currencies other than USD

**Phabricator:** https://phabricator.wikimedia.org/T365043

### Notes
* Use the USD max/min values to calculate the max value for other currencies based on the USD max of 25000
* The calculation is as follows: 
Ratio the current currency minimum donation to the USD minimum donation, then multiply this value by the max amount in USD

### Test Steps
1.  Run the app with the region set as US
2. Try to donate more than 25000 with Apple Pay
3. Verify error message
4. Set the region as any other existing in the Donate Config
5. Try to donate more than the equivalent value in USD
6. Check error message
